barretenberg/cpp/src/barretenberg/client_ivc/client_ivc.test.cpp

```diff
@@ -146,12 +146,12 @@ TEST_F(ClientIVCTests, BasicLarge)
 TEST_F(ClientIVCTests, BasicStructured)
 {
     ClientIVC ivc;
-    ivc.structured_flag = true;
+    ivc.trace_structure = TraceStructure::SMALL_TEST;
 
     // Construct some circuits of varying size
     Builder circuit_0 = create_mock_circuit(ivc, /*log2_num_gates=*/5);
-    Builder circuit_1 = create_mock_circuit(ivc, /*log2_num_gates=*/10);
-    Builder circuit_2 = create_mock_circuit(ivc, /*log2_num_gates=*/15);
+    Builder circuit_1 = create_mock_circuit(ivc, /*log2_num_gates=*/8);
+    Builder circuit_2 = create_mock_circuit(ivc, /*log2_num_gates=*/11);
```

Comment:
Nothing to gain from having these be larger values so just making them a bit smaller